pkg/controller/controller_instance.go

```diff
@@ -1116,7 +1116,7 @@ func (c *controller) processServiceInstancePollingFailureRetryTimeout(instance *
 	return c.processServiceInstancePollingTerminalFailure(instance, readyCond, failedCond)
 }
 
-// processServiceInstancePollingTerminalFailure marks the instance as having
+// processServiceInstancePollingTemporaryFailure marks the instance as having
```

Comment:
The old comment matches up with the function name, this one shouldn't be changed.  It should be 
processServiceInstancePollingTerminalFailure

Reply:
@jboyd01  I'm sorry, I have corrected it